tox_core/src/relay/server/server.rs

```diff
@@ -42,6 +46,54 @@ struct ServerState {
     pub keys_by_addr: HashMap<(IpAddr, /*port*/ u16), PublicKey>,
 }
 
+async fn send_packet(packet: Packet, mut tx: mpsc::Sender<Packet>) {
+    tokio::time::timeout(
+        TCP_SEND_TIMEOUT,
+        tx.send(packet)
+    ).await.ok();
+}
+
+/// Send all packets concurrently. The error in a single sending
+/// should not affect others.
+async fn send_packets<T: IntoIterator<Item = (Packet, mpsc::Sender<Packet>)>>(packets: T) {
+    let mut futures = packets
+        .into_iter()
+        .map(|(packet, tx)| send_packet(packet, tx))
+        .collect::<FuturesUnordered<_>>();
+    while futures.next().await.is_some() { };
+}
+
+enum HandleAction {
+    Send(Vec<(Packet, mpsc::Sender<Packet>)>),
+    Onion(OnionRequest, SocketAddr),
+}
+
+impl Default for HandleAction {
+    fn default() -> Self {
```

Comment:
impl HandleAction::ignore == HandleAction::default and use it instead.

Reply:
I'd call it `empty` then :)